test/v1/connectors/AmendOtherReliefsConnectorSpec.scala

```diff
@@ -55,18 +55,27 @@ class AmendOtherReliefsConnectorSpec extends ConnectorSpec {
 
   }
 
```

Comment:
As with the service spec, we can simplify the body by setting all the optional values to None. Also as per the service spec, we can move the values into the test trait and move the trait to the bottom of the file